### PR TITLE
Release v0.15.0

### LINKED ISSUE #1447: Release version 0.15.0
## Description

It looks like the time to release version 0.14.1, as we finished work on man pages and offline builds support, which would help package maintainers.
The only thing left unmerged, as I see, is #1417 

@rnpgp/developers  do you see any other issues which we should finish before releasing 0.14.1?

### PR DESCRIPTION
**Pull request by ni4**
Fixes #1447 

Probably we should also take a look at https://github.com/rnpgp/rnp/issues/1423 here (as per discussion https://github.com/rnpgp/rnp/issues/1447#issuecomment-794079671 ) before releasing.
CC @ronaldtse

### CODE REVIEW

**Comment by dkg** [comment]
what's the status on the 0.15.0 release?  i've been waiting for this release to drop so i can package it for debian.  Are you waiting for feedback?

If so, i can instead try uploading this branch to debian experimental as a pre-release version -- that might get you feedback from the debian infrastructure; but if you are planning to make the release soon i'd rather not do that duplicate work.

**Comment by ni4** [comment]
@dkg From my point we have only https://github.com/rnpgp/rnp/issues/1423#issuecomment-795045740 left before the release.
@ronaldtse Should we wait for the security issues email (and man updates with it and rnpgp@ribose.com) ? Or do we have other issues blocking the release?

**Comment by dewyatt** [comment]
What did we stall on here? ping @ronaldtse I guess

**Comment by ronaldtse** [comment]
@dewyatt don't think anything's stalling, it's just been busy? @ni4?

**Comment by rrrooommmaaa** [comment]
As I remember,@ni4 wanted PR #1459 to be completed in this release.
I wanted to add a couple of more tests beginning next week.

**Comment by ni4** [comment]
@dewyatt we are waiting for the 32-bit platform/time_t fixes, except that issue everything is ready.

**Comment by ronaldtse** [comment]
Thanks @ni4 , I nearly forgot!

**Comment by ni4** [comment]
All issues are resolved now, so we may finally make a release. Please let me merge this once approved as I need to do some extra steps afterwards.

**Comment by rrrooommmaaa** [comment]
We're having some intermittent issues with automake-1.16 missing on i386/Debian9, have you seen them?
As I understand there is 1.15-1 in Debian's repository and we need it to build gpg, but sometimes the build succeeds

**Comment by ni4** [comment]
Yeah, I saw it on this PR. However after re-running it succeeded. So it should be related to runner configuration, not to the RNP itself. If it will appear more frequent we may fix it via additional issue.

File: CHANGELOG.md
Comment (antonsviridenko) [inline CHANGELOG.md:9]: "whitespace characters" would be more correct phrasing

Comment (ni4) [reply CHANGELOG.md]: @antonsviridenko Thanks, fixed.

**Review by antonsviridenko — APPROVED** [review]

**Comment by ronaldtse** [comment]
The debian build fails on the lack of the `automake` command:

```
/__w/rnp/rnp/builds/gpg/libgpg-error-1.39/build-aux/missing: line 81: automake-1.16: command not found
1113
WARNING: 'automake-1.16' is missing on your system.
1114
```

**Review by ronaldtse — APPROVED** [review]

**Comment by ni4** [comment]
@ronaldtse After restarting debian build succeeds, so merging.

**Comment by ronaldtse** [comment]
Congratulations to all!! 🎉 🎆 👏

**Pull request merged**
